reference/orders.v3.yml

```diff
@@ -2909,22 +2786,78 @@ components:
           description: |
             The type of resource with which the metafield is associated.
           enum:
-            - category
             - order
             - brand
             - product
             - variant
+            - category
+          example: order
         resource_id:
-          type: string
+          type: integer
           description: |
             The unique identifier for the resource with which the metafield is associated.
-          maxLength: 10000000000
+          example: 0
```

Comment:
🍹 : may use `1` ? not sure if `0` is a valid resource id

Reply:
That works, thanks!